helm/chart/router/README.md

```diff
@@ -70,7 +70,7 @@ helm show values apollographql/router
 | resources | object | `{}` |  |
 | rhai | object | `{"input_file":""}` | If using rhai, specify the location of your input file |
 | rhai.input_file | string | `""` | input rhai file, contents will be stored in a ConfigMap |
-| router | object | `{"args":["--hot-reload"],"configuration":{"supergraph":{"listen":"0.0.0.0:80"}}}` | See https://www.apollographql.com/docs/router/configuration/overview#configuration-file for yaml structure |
+| router | object | `{"args":["--hot-reload"],"configuration":{"graphql":{"listen":"0.0.0.0:80"}}}` | See https://www.apollographql.com/docs/router/configuration/overview#configuration-file for yaml structure |
```

Comment:
Shouldn't that be supergraph, not graphql?

Reply:
[2427dcc](https://github.com/apollographql/router/pull/1859/commits/2427dcce26ca0dc82dda80fbc376e6832f282955)